test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1982,6 +1982,35 @@ static public void testDefaultValueInPointArray() throws Throwable {
 		}
 	}
 
+	/**
+	 * Create multianewarray with Point Class without initialization.
+	 * The array should be set ot a Default Value.
+	 */
+	@Test(priority=4)
+	static public void testDefaultValueInPointInstanceMultiANewArray() throws Throwable {
+		Object pointArray = Array.newInstance(point2DClass, new int[]{genericArraySize, genericArraySize});
+		for (int i = 0; i < genericArraySize; i++) {
+			for (int j = 0; j < genericArraySize; j++) {
+				Object pointObject = Array.get(Array.get(pointArray,i),j);
+				assertNotNull(pointObject);
+			}
+		}
+	}
+
+	@Test(priority=4)
+	static public void testDefaultValueInPointByteCodeMultiANewArray() throws Throwable {
+		/* Try down cast? ELSE: look at ASMifier */
+		MethodHandle makePointArray = lookup.findStatic(point2DClass, "generate2DMultiANewArray", MethodType.methodType(Object.class, int.class, int.class));
+		Object pointArray = makePoint2D.invoke(genericArraySize, genericArraySize);
+		MethodHandle getPoint2D = lookup.findStatic(point2DClass, "get2DMultiANewArray", MethodType.methodType(Object.class, Object.class, int.class, int.class));
+		//for (int i = 0; i < genericArraySize; i++) {
+			//for (int j = 0; j < genericArraySize; j++) {
+				Object pointObject = getPoint2D.invoke(pointArray,1, 1);
```

Comment:
also you dont need to generate code for getters and setters, since Array.get will do the same thing